### ISSUE TITLE
Calling Julia function from Python Threads runs for forever

### ISSUE THREAD

**Issue by JulesBlubb** [JuliaPy/PyCall.jl#765]
I am importing a Python script into my Julia code. I am then trying to call a Julia function within a Multi-threading Python process, which leads to a Deadlock. How can I adapt the Julia Code to be able to call Python with Multi-threading?

My example code:
https://github.com/JulesBlubb/Julia-Multithreading-Problem

System Details:
- Ubuntu 19.10
- Julia 1.3.1 with Pycall v1.91.4
- Python: 3.6.5
 
Thank you.

**Comment by tkf** [comment]
I don't think it is possible to call into `julia` via C API from different threads. That's the limitation of `julia`, not PyCall. Also, even if it were possible, it is not beneficial/possible to use PyCall with multi-threaded Python code since PyCall does not release the GIL.
